Patches the blue inverted color when using the decon driver

---
diff --git a/drivers/video/fbdev/exynos/dpu30/decon_dsi.c b/drivers/video/fbdev/exynos/dpu30/decon_dsi.c
index 117cd0c31c5a..f3d660dbf26b 100644
--- a/drivers/video/fbdev/exynos/dpu30/decon_dsi.c
+++ b/drivers/video/fbdev/exynos/dpu30/decon_dsi.c
@@ -1097,11 +1097,11 @@ int decon_check_var(struct fb_var_screeninfo *var, struct fb_info *info)
 	case 24:
 		/* our 24bpp is unpacked, so 32bpp */
 		var->bits_per_pixel	= 32;
-		var->red.offset		= 16;
+		var->red.offset		= 0;
 		var->red.length		= 8;
 		var->green.offset	= 8;
 		var->green.length	= 8;
-		var->blue.offset	= 0;
+		var->blue.offset	= 16;
 		var->blue.length	= 8;
 		break;
 
